src/txmempool.cpp

```diff
@@ -663,15 +663,19 @@ void CTxMemPool::removeForReorg(CChainState& active_chainstate, int flags)
                 }
             }
         }
-        if (!validLP) {
-            mapTx.modify(it, update_lock_points(lp));
-        }
     }
     setEntries setAllRemoves;
     for (txiter it : txToRemove) {
         CalculateDescendants(it, setAllRemoves);
     }
     RemoveStaged(setAllRemoves, false, MemPoolRemovalReason::REORG);
+    auto chain = active_chainstate.m_chain;
```

Comment:
bedf246f1e2497a3641093c6e8fa11fb34dddac4:

Is this creating a copy of a vector holding 1kk elements for no reason?

Reply:
woop, should have declared as `auto&`. The good thing is it's removed in the last commit?